From f2c29e43142f32bafc33a9031aa0c50e31651113 Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Wed, 10 Apr 2019 14:50:47 +0300
Subject: [PATCH 689/706] arm64: dts: ls1028a: chmod -x on dts files

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
(cherry picked from commit d7693fda9c2a0db53fdbd7db81ba017d95d217bd)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts | 0
 arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts | 0
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi    | 0
 3 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
 mode change 100755 => 100644 arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
 mode change 100755 => 100644 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
old mode 100755
new mode 100644
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
old mode 100755
new mode 100644
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
old mode 100755
new mode 100644
-- 
2.17.1

